sovtokenfees/sovtokenfees/test/taa/xfer/test_taa_xfer.py

```diff
@@ -0,0 +1,74 @@
+import json
+import time
+
```

Comment:
Please add integration tests for txn with FEEs and TAA. This is needed to make sure, that TAA fields are not included when verifying payment signature for FEEs.
The tests may be skipped for now.

1. Test 1:
- Enable (set) TAA
- Send NYM with FEEs and TAA fields
- Make sure it's written fine
2. Test 2:
- Do not enable TAA (do not set it)
- Send NYM with FEEs and TAA fields
- Make sure it's written fine
3. Test 3:
- Enable (set) TAA
- Send NYM with FEEs and without TAA fields
- Make sure it's Rejected with with missing TAA
4. Test 4:
- Do not enable TAA (do not set it)
- Send NYM with FEEs and without TAA fields
- Make sure it's written fine